lib/suspenders/app_builder.rb

```diff
@@ -467,6 +424,10 @@ def configure_rubocop
       template '.rubocop.yml', '.rubocop.yml'
     end
 
+    def run_stairs
+      bundle_command 'exec stairs --use-defaults'
```

Comment:
each task has the opportunity to bundle. at this point, during generation `bundle install` happens more than a few times. I imagine its a problem that will only get worse.